test failure on windows

testSetupWithGoogleSettingSdkConfigured()


```
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
Unexpected exception thrown.
org.gradle.messaging.remote.internal.MessageIOException: Could not write message [EndOfStream] to '/127.0.0.1:55726'.
	at org.gradle.messaging.remote.internal.inet.SocketConnection.dispatch(SocketConnection.java:111)
	at org.gradle.messaging.remote.internal.hub.MessageHub$ConnectionDispatch.run(MessageHub.java:284)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:54)
	at org.gradle.internal.concurrent.StoppableExecutorImpl$1.run(StoppableExecutorImpl.java:40)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.io.IOException: An existing connection was forcibly closed by the remote host
	at sun.nio.ch.SocketDispatcher.write0(Native Method)
	at sun.nio.ch.SocketDispatcher.write(SocketDispatcher.java:51)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:93)
	at sun.nio.ch.IOUtil.write(IOUtil.java:51)
	at sun.nio.ch.SocketChannelImpl.write(SocketChannelImpl.java:471)
	at org.gradle.messaging.remote.internal.inet.SocketConnection$SocketOutputStream.writeWithNonBlockingRetry(SocketConnection.java:248)
	at org.gradle.messaging.remote.internal.inet.SocketConnection$SocketOutputStream.writeBufferToChannel(SocketConnection.java:236)
	at org.gradle.messaging.remote.internal.inet.SocketConnection$SocketOutputStream.flush(SocketConnection.java:230)
	at org.gradle.messaging.remote.internal.inet.SocketConnection.dispatch(SocketConnection.java:109)
	... 6 more
:google-cloud-tools-plugin:test FAILED
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':google-cloud-tools-plugin:test'.
> Process 'Gradle Test Executor 3' finished with non-zero exit value -1073740791
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
BUILD FAILED
Total time: 1 mins 15.399 secs
Process 'Gradle Test Executor 3' finished with non-zero exit value -1073740791
```

--- a/google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/project/DefaultAppEngineProjectService.java
+++ b/google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/project/DefaultAppEngineProjectService.java
@@ -254,9 +254,7 @@ public class DefaultAppEngineProjectService extends AppEngineProjectService {
       return Optional.empty();
     }
 
-    try (
-        BufferedReader reader = Files.newBufferedReader(appYamlPath, Charset.defaultCharset());
-    ) {
+    try (BufferedReader reader = Files.newBufferedReader(appYamlPath, Charset.defaultCharset())) {
       Object parseResult = yamlParser.load(reader);
 
       if (!(parseResult instanceof Map)) {

--- a/google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/project/DefaultAppEngineProjectService.java
+++ b/google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/project/DefaultAppEngineProjectService.java
@@ -42,6 +42,7 @@ import org.jetbrains.plugins.gradle.util.GradleConstants;
 import org.yaml.snakeyaml.Yaml;
 import org.yaml.snakeyaml.scanner.ScannerException;
 
+import java.io.BufferedReader;
 import java.io.IOException;
 import java.nio.charset.Charset;
 import java.nio.file.Files;
@@ -247,14 +248,16 @@ public class DefaultAppEngineProjectService extends AppEngineProjectService {
   private Optional<String> getValueFromAppYaml(@NotNull String appYamlPathString,
       @NotNull String key) throws MalformedYamlFileException {
     Yaml yamlParser = new Yaml();
-    try {
-      Path appYamlPath = Paths.get(appYamlPathString);
-      if (!Files.isRegularFile(appYamlPath)) {
-        return Optional.empty();
-      }
 
-      Object parseResult =
-          yamlParser.load(Files.newBufferedReader(appYamlPath, Charset.defaultCharset()));
+    Path appYamlPath = Paths.get(appYamlPathString);
+    if (!Files.isRegularFile(appYamlPath)) {
+      return Optional.empty();
+    }
+
+    try (
+        BufferedReader reader = Files.newBufferedReader(appYamlPath, Charset.defaultCharset());
+    ) {
+      Object parseResult = yamlParser.load(reader);
 
       if (!(parseResult instanceof Map)) {
         return Optional.empty();

